seed/celery.py

```diff
@@ -31,7 +31,7 @@ def on_configure(self):
             pass
 
 
-app = Celery('seed')
+app = Celery('seed', broker='redis://db-redis:6379/', backend='redis://db-redis:6379/')
```

Comment:
backend undefined unless set on app.

Celery config before including `backend` 
```
seed_celery    |  -------------- celery@ac196d7f2f6a v5.2.2 (dawn-chorus)
seed_celery    | --- ***** -----
seed_celery    | -- ******* ---- Linux-5.10.47-linuxkit-x86_64-with 2022-01-25 18:59:49
seed_celery    | - *** --- * ---
seed_celery    | - ** ---------- [config]
seed_celery    | - ** ---------- .> app:         seed:0x7f1a8b559a60
seed_celery    | - ** ---------- .> transport:   redis://db-redis:6379//
seed_celery    | - ** ---------- .> results:     disabled://
seed_celery    | - *** --- * --- .> concurrency: 2 (prefork)
seed_celery    | -- ******* ---- .> task events: ON
seed_celery    | --- ***** -----
seed_celery    |  -------------- [queues]
seed_celery    |                 .> celery           exchange=celery(direct) key=celery
```

Reply:
Hm, I'm not sure if this will work when we deploy on k8s... @nllong thoughts? If the hostname is different there then this will be an issue. Maybe we use an env var?
Or maybe like what we're doing here?
https://github.com/SEED-platform/seed/pull/3110/files#diff-35b09269a5cadc5c19631d6a33576850205875136786104647845d094c928483R35

Reply:
oooh, good point! Yes, it should pull from the env vars. BUT, I think db-redis will work just fine in k8s because that is the name of container across any number of pods.